docs/_posts/2015-07-23-new-react-devtools-beta.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "New React Devtools Beta"
+author: jaredly
+---
+
+We've made an entirely new version of the devtools, and we want you to try it
+out!
+
+![The full devtools gif](/react/img/blog/devtools-full.gif)
+
+## Why entirely new?
+
+Perhaps the biggest reason was to create a defined API for dealing with
+internals, so that other tools could benefit as well and not have to depend on
+implementation details. This gives us more freedom to refactor things
+internally without worrying about breaking tooling.
+
+The current version of the devtools is a fork of blink's "Elements" pane, and
+is imperative, mutation-driven, and tightly integrated with chrome-specific
+APIs. The new devtools are much less coupled to chrome, and easier to reason
+about&mdash;thanks to React.
```

Comment:
Don't need to use html entities, can just put the actual — in there. But to be nitpicky, I think an em dash is wrong here. You split the thought with a comma and rejoin with the dash. One or the other :)